lang/rust/rust.talon

```diff
@@ -64,6 +64,7 @@ state (some | sum): "Some"
 state static: "static "
 self taught: "self."
 state use: user.code_import()
+state fish | state turbo fish | state turbo: user.insert_between("::<", ">")
```

Comment:
I'm not a Rust expert, but we usually try to keep the proliferation of alternative pronunciations to a minimum. Would `state [turbo] fish` be acceptable to Rust programmers?